src/ovirtapi/index.js

```diff
@@ -164,7 +165,7 @@ const OvirtApi = {
     return httpGet({ url })
   },
 
-  addNewVm ({ vm, transformInput = false }: VmType): Promise<Object> {
+  addNewVm ({ vm, transformInput = false }: { vm: Object, transformInput?: boolean }): Promise<Object> {
```

Comment:
I think here must `ApiVmType` instead of `Object`.

Reply:
I think this got out-of-sync with the same change on the Details PR #701.

At this point, and in general, I think the inputs should by the internal type or `Object`.  There are plenty of times a rest action only needs a partial object.  The returns we should leave as `Object` for now as the return typing needs a bit more thought.

All that being said, I'll update the function's type to:
````js
addNewVm ({ vm, transformInput = false }: { vm: VmType | Object, transformInput: boolean }): Promise<Object> {
````